docs/serving/knative-kubernetes-services.md

```diff
@@ -82,10 +84,9 @@ and validates and mutates Kubernetes API calls.
 
 The certmanager reconciles cluster ingresses into cert manager objects.
 
-### Deployment: networking-istio
+### Deployment: networking-istio/contour-ingress-controller/etc.
 
-The networking-istio deployment reconciles a cluster's ingress into an
-[Istio virtual service](https://istio.io/docs/reference/config/networking/v1alpha3/virtual-service/).
+The networking layer deployment (eg. networking-istio, contour-ingress-controller, etc.) reconciles a cluster’s ingress into the appropriate networking layer primitives. In the example shown above, we've chosen to use Istio (which reconciles into a [Istio Virtual Service](https://istio.io/latest/docs/reference/config/networking/virtual-service/), but any networking layer deployment will do.
```

Comment:
nit: The last snippet `, but any networking layer deployment will do` could be dropped as the operator doesn't choose the networking layer deployment but chooses a networking layer.